### PR TITLE
implemented usd/currency with localstorage save

### PR DESCRIPTION
**Pull request by nthgness**
# Pull request template

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [ ] branch rebased on `master`
- [ ] styleguide check `npm run validate`
- [ ] good naming
- [ ] keep simple
- [ ] video or screenshot attached
- [ ] testing instruction attached
- [ ] check your PR once again

### What and how to test

To test it locally run ```git fetch && fit checkout twelveWordsCreateBug``` where twelveWordsCreateBug is a name of this branch (see under title) OR ```git fetch origin refs/pull/2590/merge:pr2590 && git checkout pr2590``` where 2590 is number of this pull request

```npm i && npm run build:mainnet``` then run MultiCurrencyWallet/build.mainnet/index.html in browse

### CODE REVIEW

File: shared/components/modals/WithdrawModal/WithdrawModal.js [hunk 1]
```diff
@@ -740,43 +740,29 @@ export default class WithdrawModal extends React.Component {
...
+              onClick={() => {
+                this.setState({ selectedValue: 'USD' })
+                localStorage.setItem(constants.localStorage.balanceActiveCurrency, 'usd')
+              }}
+            >
+              USD
+            </span>
+            <span styleName="delimiter"></span>
+            <span
+              styleName={cx('additionalСurrenciesItem', { additionalСurrenciesItemActive: selectedValue === currentActiveAsset.currency })}
```
Comment (Pasha8914) [inline shared/components/modals/WithdrawModal/WithdrawModal.js:755]: вынеси в функцию, передавай агрументы для нового значения `selectedValue ` и `additionalСurrenciesItemActive` ургументами

Comment (nthgness) [reply shared/components/modals/WithdrawModal/WithdrawModal.js:755]: принял

**Comment by nthgness** [comment]
https://drive.google.com/file/d/1SzNjlkObzbKm-xVTB8lwLCk3amXpMzxU/view

**Comment by noxonsu** [comment]
- [ ] ошибки не красиво https://screenshots.wpmix.net/chrome_D4QM3pRYW9IncSdPaxLtisIUWKfH3qYh.png

**Pull request merged**